ocaml/xapi/xapi_globs.ml

```diff
@@ -917,6 +917,8 @@ let winbind_machine_pwd_timeout = ref (7 * 24 * 3600)
 
 let winbind_update_closest_kdc_interval = ref (3600. *. 24.) (* every day *)
 
+let winbind_kerberos_encryption_type = ref "strong"
```

Comment:
Why are we strong a string and not the proper type?

Reply:
I think the configuration part only take care of the configurations, and let the logic part (winbind) take care of the types.

Reply:
By using the proper type you need to validate the incoming string only once. Now anything could be in that string and any client needs to validate it